packages/trigger-client/src/dataMappers.ts

```diff
@@ -91,7 +91,7 @@ export function mapServerOrderInfo(
     amount: toBigDecimal(serverOrder.order.amount),
     expiration: toBigDecimal(serverOrder.order.expiration),
     nonce: serverOrder.order.nonce,
-    price: fromX18(serverOrder.order.priceX18),
+    price: removeDecimals(toBigDecimal(serverOrder.order.priceX18)),
```

Comment:
Is the `toBigDecimal` needed?

Reply:
Yeah, needs casting because `bigDecimalish` could be either `number | BigNumber`.